disperser/cmd/controller/main.go

```diff
@@ -201,5 +224,54 @@ func RunController(ctx *cli.Context) error {
 		}
 	}()
 
+	// Create readiness probe file once the controller starts successfully
+	if _, err := os.Create(controllerReadinessProbePath); err != nil {
+		logger.Warn("Failed to create readiness file", "error", err, "path", controllerReadinessProbePath)
+	}
+
+	// Signal heartbeat
+	signalHeartbeat(logger)
```

Comment:
We should add heartbeat signals in the main processing loops of both components:
1. Encoding manager: before `HandleBatch()`
2. Dispatcher: before `HandleBatch()`

Right now we're only signaling liveness at startup. But what we want is to have heartbeats for each control loop. Then our alerting system should alert if any of the two control loops fail and we should know which one failed.